xtask/src/openapi.rs

```diff
@@ -0,0 +1,63 @@
+use std::path::Path;
+use std::process::Command;
+use std::{env, fs};
+
+use anyhow::anyhow;
+use clap::Parser;
+
+use trustify_server::openapi::openapi;
+
+#[derive(Debug, Parser)]
+pub struct Validate {}
+
+impl Validate {
+    pub fn run(self) -> anyhow::Result<()> {
+        let command = if command_exists("podman") {
+            "podman"
+        } else if command_exists("docker") {
+            "docker"
+        } else {
+            return Err(anyhow!("Neither podman nor docker is installed"));
```

Comment:
Alternately you could use more assertive language, for example "This task requires podman or docker to be installed.".